### PR TITLE
[FIX] MediaInfo not showing up on the platform

### LINKED ISSUE #507: [BUG] Not all mediainfo is showing up on the site
Sample platform commit (found at the bottom of each page) : 8a90a9cf92

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [x] I have checked that the issue I'm posting isn't already reported.
- [x] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/CCExtractor/sample-platform/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/CCExtractor/sample-platform/issues)
- [x] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [] I have never visited/used the platform.
- [] I have used the platform just a couple of times.
- [] I have used the platform extensively, but have not contributed previously.
- [x] I am an active contributor to the platform.

---

Some recent samples (i.e. https://sampleplatform.ccextractor.org/sample/164) are not showing media info while the xml file is available on the repository. I suspect a parsing issue is at hand. 

The failing xml is attached: [failing.xml.zip](https://github.com/CCExtractor/sample-platform/files/6096148/failing.xml.zip)

### PR DESCRIPTION
**Pull request by kvshravan**
**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

---

Fixes #507

### CODE REVIEW

File: mod_sample/media_info_parser.py [hunk 1]
```diff
@@ -212,13 +215,9 @@ def generate_media_xml(sample) -> MediaInfoFetcher:
...
         process.wait()
         if os.path.isfile(media_info_path):
-            # Load media info, and replace full pathname
+            # Load media info
             tree = etree.parse(media_info_path)
-            for elem in tree.xpath("//track[@type='General']"):
-                for child in elem:
-                    if child.tag == 'Complete_name':
-                        child.text = child.text.replace(media_folder, '')
-                        break
```
Comment (canihavesomecoffee) [inline mod_sample/media_info_parser.py:221]: Why was this deleted? We don't want to leak path names of the file to persons who download the XML.

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

**Comment by canihavesomecoffee** [comment]
You'll also have to update the unit tests ;)

File: mod_sample/media_info_parser.py (hunk 1 above)
Comment (kvshravan) [reply mod_sample/media_info_parser.py:221]: There is no such tag as "Complete_name" in the XML generated by the latest version of mediainfo.

File: mod_sample/media_info_parser.py [hunk 2]
```diff
@@ -214,11 +217,13 @@ def generate_media_xml(sample) -> MediaInfoFetcher:
...
+            root = tree.getroot()
+            try:
+                for rootElems in root:
+                    if rootElems.tag.split('}')[1] == 'media':
```
Comment (kvshravan) [inline mod_sample/media_info_parser.py:223]: There is no such tag as `Complete_name` in the XML generated by the latest version of mediainfo, but there is a tag `"media"` which has an attribute `ref` containing full path name of `media_folder`. When looped through the tags , every tag is prefixed with `'{https://mediaarea.net/mediainfo}'`, say for example the tag `media` when printed will look something like this `'{https://mediaarea.net/mediainfo}media'` , the split function is used to split the prefix and tag.

File: mod_sample/media_info_parser.py (hunk 1 above)
Comment (canihavesomecoffee) [reply mod_sample/media_info_parser.py:221]: It's now located in the Media ref attribute. The thing we want to achieve is replacing the full filename (including directories) with just the filename.

Comment (kvshravan) [reply mod_sample/media_info_parser.py:221]: Yes, I just made a comment on that in the below part of the code :)

File: mod_sample/media_info_parser.py (hunk 2 above)
Comment (canihavesomecoffee) [reply mod_sample/media_info_parser.py:223]: I wonder if the prefixes can gotten rid of by loading the XSD in some way, but that's not that important to fix.

Comment (kvshravan) [reply mod_sample/media_info_parser.py:223]: Yes, [localname](https://stackoverflow.com/questions/18159221/remove-namespace-and-prefix-from-xml-in-python-using-lxml) can be used to eliminate those prefixes.

Comment (canihavesomecoffee) [reply mod_sample/media_info_parser.py:223]: That'd look a bit neater I'd say.

Don't forget about the unittests, they're still failing.

Comment (kvshravan) [reply mod_sample/media_info_parser.py]: Yes I'll update them :)

**Review by canihavesomecoffee — APPROVED** [review]
